src/features/courses/course-route-validation.ts

```diff
@@ -1,12 +1,13 @@
 import * as Joi from '@hapi/joi';
+import { getEnumValues } from '../../utilities/utilities';
+import { ListCoursesFilters } from './course-controller';
 
 export const createCourseValidation = {
     params: {},
-    query: {
```

Comment:
Why is this removed?
The idea here is that we could say that we wanted a course to reference a curriculum or a course without actually deep copying